src/main/java/org/javarosa/xpath/XPathTypeMismatchException.java

```diff
@@ -22,6 +22,6 @@ public XPathTypeMismatchException() {
     }
 
     public XPathTypeMismatchException(String s) {
-        super("type mismatch \n" + s);
```

Comment:
Was the thinking here that "type mismatch" was not useful information? Seems like we might want to keep some part of the message as indicative that this came from this exception class